db/post_migrate/20200430173433_remove_user_id_from_invites.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RemoveUserIdFromInvites < ActiveRecord::Migration[5.2]
+  def up
+    execute <<~SQL
+      INSERT INTO invited_users (
+        user_id,
+        invite_id,
+        redeemed_at,
+        created_at,
+        updated_at
+      )
+      SELECT user_id, id, redeemed_at, created_at, updated_at
+      FROM invites WHERE user_id IS NOT NULL
+    SQL
+
+    execute <<~SQL
+      ALTER TABLE invites DROP COLUMN IF EXISTS user_id
```

Comment:
If we don't drop this column, will there be any side effects? Because this migration is irreversible, I'll recommend keeping the column around first as backup until we're confident everything is working well.

Reply:
No side effects. Makes sense. I've made the required changes.